test/unit/unit_deprecate_warning.test.js

```diff
@@ -49,30 +48,15 @@ describe('Deprecation Warnings - unit', function () {
       f2({ maxScan: 5 });
     });
 
-    it('multiple functions with the same deprecated options should both warn', {
-      metadata: { requires: { node: '>=6.0.0' } },
-      test: function (done) {
-        process.nextTick(() => {
-          expect(messages).to.deep.equal([
-            'f1 option [maxScan]' + defaultMessage,
-            'f2 option [maxScan]' + defaultMessage
-          ]);
-          expect(messages).to.have.a.lengthOf(2);
-          done();
-        });
-      }
-    });
-
-    it('multiple functions with the same deprecated options should both warn', {
-      metadata: { requires: { node: '<6.0.0' } },
-      test: function (done) {
-        ensureCalledWith(console.error, [
+    it('multiple functions with the same deprecated options should both warn', done => {
```

Comment:
I'd remove the `unit_` prefix from the filename as it's a bit redundant already being in the `unit` directory.

Reply:
Good catch, I missed this change.  Thanks!